tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -355,6 +363,19 @@ function check_dismiss_changelog_review($payload){
 				dismiss_review($payload, $R['id'], 'Changelog added/fixed.');
 }
 
+function close_if_other_feature_prs($payload){
+	$singleton_labels = array('Feature', 'Balance/Rebalance');
+
+	$query = array('is' => array('pr', 'open'), 'author' => $payload['pull_request']['user']['login'], 'label' => $singleton_labels);
+	$res = issue_search($query);
+	if(count($res) < 2)
+		return FALSE;
```

Comment:
This still has the problem that you're counting prs that have a positive value as negative.

You need to make it tag negative value prs as negative, and positive value prs as positive, so you can check that here rather than snowflake this check for tags without accounting for any other tags that might raise it above a negative value

Reply:
You also are counting prs with `no update` tags when they should be ignored.